platform-hub-api/app/serializers/identity_serializer.rb

```diff
@@ -8,4 +8,11 @@ class IdentitySerializer < ActiveModel::Serializer
     :created_at,
     :updated_at
   )
+
+  attribute :kubernetes_tokens, if: -> { object.provider.to_sym == :kubernetes }
+
+  def kubernetes_tokens
+    tokens = Kubernetes::TokenService.tokens_from_identity(object)
+    ActiveModel::Serializer::CollectionSerializer.new(tokens, each_serializer: KubernetesTokenSerializer)
+  end
```

Comment:
Moved kubernetes_tokens method into a block. For some reason serializer / each_serializer wouldn't work when passed as attribute arguments. Leaving rest as is.